kimchi/src/tests/foreign_field_mul.rs

```diff
@@ -1196,15 +1110,15 @@ fn test_random_multiplicands_carry1_lo() {
             &left_input,
             &right_input,
             &secp256k1_modulus(),
-            vec![((0, 4), PallasField::one())], // Invalidate carry1_lo
+            vec![((0, 6), PallasField::one())], // Invalidate carry1_lo
         );
         assert_eq!(
             (&left_input * &right_input) % secp256k1_modulus(),
             [witness[0][1], witness[1][1], witness[2][1]].compose()
         );
         assert_eq!(
             result,
-            Err(CircuitGateError::Constraint(GateType::ForeignFieldMul, 2)),
+            Err(CircuitGateError::Constraint(GateType::ForeignFieldMul, 5)),
```

Comment:
Also, it doesn't seem like you changed order of the constraints so much so that it goes from 2 to 5 in one single change.

Reply:
Previous test was bad.  5 is the constraint that should be impacted by an invalid `carry1_lo`